etna/transforms/missing_values/resample.py

```diff
@@ -44,13 +44,19 @@ def _get_folds(self, df: pd.DataFrame) -> List[int]:
         Here the ``in_column`` frequency gap is divided into the folds with the size of dataset frequency gap.
         """
         in_column_index = df[self.in_column].dropna().index
-        if len(in_column_index) <= 1 or (len(in_column_index) >= 3 and not pd.infer_freq(in_column_index)):
+
+        fail_datetime_timestamp = (
```

Comment:
Looks like we can rewrite it in more general way (what if we have the other type of index?) and convert to some utils function which will infer freq from dataframe -- we won't do it here, just idea for the future

Reply:
Ok, may be we'll do smth like this in the future.